frontend/JavaScript/footer.js

```diff
@@ -0,0 +1,56 @@
+function fetchSortKey(){
+    selector = document.querySelector("#sort");
+    sortKey = selector.value;
+    return sortKey;
+}
+
+function home(){
+    window.parent.location=`index.html?q=*`;
+  }
+    
+
+function prevPage(){
+    const queryString = window.location.search;
+    const urlParams = new URLSearchParams(queryString);
+    prod_query =urlParams.get('q');
+    var sortKey = urlParams.get('sort');
+    catLevel1 = urlParams.get('cat1');
+    catLevel2 = urlParams.get('cat2');
+    if ((sortKey==null)||(sortKey=='null'||sortKey=="")){
```

Comment:
Use "===" for comparing , that will check value as well as datatype

Reply:
"===" has replaced "==" and "!==" has replaced "!=" for comparison.